src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -1658,17 +1658,28 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		return connections;
 	}
 
-	async handleUnsupportedProvider(providerName: string): Promise<boolean> {
-		const extensionId = ConnectionProviderAndExtensionMap.get(providerName);
+	public async handleUnsupportedProvider(providerId: string): Promise<boolean> {
+		const extensionId = ConnectionProviderAndExtensionMap.get(providerId);
```

Comment:
Also - note that we could potentially do away with this hardcoded mapping at some point in the future. We'd just need to use the extension gallery service to get the extension metadata, pull down the package.json and then inspect it for contributed providers. But that's a lot of work for something that isn't really necessary at this time so what we have is fine IMO.